template.php

```diff
@@ -140,12 +140,18 @@ function UTKdrupal_preprocess_page(&$variables, $hook) {
       unset($variables['page']['content']['system_main']['citation.tab']['citation']['#markup']);
       }
       $thisDetails = $variables['page']['content']['system_main']['citation.tab']['metadata']['#markup'];
+      //Revise for Author display
       $Author = extractStringValue($thisDetails,'utk_mods_etd_name_author_ms');
-      $prefix  = '<div class="csl-bib-body"><div class="citation_author_container"><div class="citation_author"><h4>';
+      $prefix  = '<div class="csl-bib-body"><div class="citation_author_container"><h>4 class="citation_author">';
```

Comment:
Looks like you have a typo <h>4 class="citation_author">

Reply:
Due to this problem with the <h4> tag which I have "fixed" and "pushed" more than once and several other problems that keep returning, I have determined that this branch is out of sync with horizontal (due to several merges).  I will close this pull request, delete this branch, and make a new branch to take care of TRAC-1022.